main.py

```
@@ -1,285 +1,223 @@
-"""
-SLAG Commenting System - FastAPI Backend
-A commenting system based on storing JSON files in a directory structure.
-"""
+from fastapi import FastAPI, HTTPException, Request, Body
```

Ruff (0.11.9)

`fastapi.Request` imported but unused

Remove unused import

(F401)